content/docs/guides/continuous-delivery/spinnaker.md

```diff
@@ -1,77 +1,102 @@
-
-**Note**: The names used above are purely for demonstration purposes only.
-You may choose a naming convention that best suits your organization.
+The example in this topic shows you how to deploy a Kubernetes resource; however, you can use any of the other
+[cloud providers](https://www.pulumi.com/docs/intro/cloud-providers/) that Pulumi supports.
 
 ## Pulumi Plugin For Spinnaker
 
-The Pulumi plugin executes a pre-configured job as a Kubernetes batch job. The job runs whichever container image you choose to use for executing the Pulumi apps. The only requirement for the image that the batch job uses is that it should contain the Pulumi CLI, and any necessary language runtime tools, such as `node`, `python`, `dotnet`, or `go`.
+The Pulumi plugin executes a preconfigured job as a Kubernetes batch job. The job uses the container image you specify
+in the Spinnaker pipeline UI.
+
 By default, the plugin suggests using the [`pulumi/pulumi`](https://hub.docker.com/r/pulumi/pulumi) container image available publicly on Docker Hub.
 Alternatively, you can also try to use one of the other language-specific images that are available under the [`pulumi`](https://hub.docker.com/u/pulumi) org on Docker Hub.
 
-If you would like to use a private image for the job you can do so, but ensure that your Spinnaker cluster has the right credentials to pull an image from a private registry.
-Contact your Spinnaker administrator if you are unsure whether you can use a public image.
+If you would like to use your own container image for the job, ensure that the following requirements are met:
 
-The pre-configured job will also need to access your VCS to clone the repo into the container. See the next section about configuring secrets to learn how you can use private repos with the plugin.
+* Your container should include the Pulumi CLI, and any necessary language runtime tools, such as `node`, `python`, `dotnet`, or `go`.
+* If your container image is in a private registry, your Spinnaker cluster must have the proper credentials to pull from the registry.
+
+The preconfigured job will also need to access your VCS to clone the repo into the container.
+See the next section about configuring secrets to learn how you can use private VCS repos with the plugin.
 
 ### Secrets
 
-To pass sensitive secrets to the Kubernetes batch job run by the plugin, you'll need to define a [Kubernetes Secret](https://kubernetes.io/docs/concepts/configuration/secret/). The secret will need to be in the same namespace as the namespace used to run the pre-configured job.
-Sensitive secrets such as the Pulumi Access Token, the cloud provider credentials, or even your Git VCS credentials (eg. GitHub, Bitbucket, GitLab), should be defined as key/value pairs in the Secret resource.
+To pass sensitive secrets to the Kubernetes batch job run by the plugin, define a [Kubernetes `Secret`](https://kubernetes.io/docs/concepts/configuration/secret/).
+
+{{% notes "warning" %}}
```

Comment:
Hmm this still renders as an informational note only. Does a "warning" note exist in our templates?

Reply:
I assume so, unless I'm interpreting the notes shortcode incorrectly. https://github.com/pulumi/docs/blob/master/layouts/shortcodes/notes.html#L2

Reply:
Nevermind. The type needs to be explicitly set. I just checked this in my local branch. (Same for the other warnings above.) Specifying anything else (or nothing at all) will result in a note.

```suggestion
{{% notes type="warning" %}}
```